engine_modules/corporation/tests/test_models.py

```diff
@@ -29,30 +31,17 @@ def test_corporation_update_assets(self):
 		self.assertEqual(self.reload(self.c).assets, initial_corporation_assets - 1)
 		self.assertEqual(self.reload(corporation_market).value, initial_market_assets - 1)
 
-	def test_corporation_update_assets_in_historic_market(self):
-		"""
-		Corporation assets should be updated in historic market by default
-		"""
-
-		corporation_historic_market = self.c.historic_corporation_market
-		initial_corporation_assets = self.c.assets
-		initial_market_assets = corporation_historic_market.value
-
-		self.c.update_assets(-1, category=AssetDelta.EFFECT_FIRST)
-
-		self.assertEqual(self.reload(self.c).assets, initial_corporation_assets - 1)
-		self.assertEqual(self.reload(corporation_historic_market).value, initial_market_assets - 1)
-
 	def test_corporation_update_assets_not_below_zero(self):
 		"""
 		Corporation market assets can't drop below 0
 		"""
 
-		corporation_historic_market = self.c.historic_corporation_market
+		c_markets = self.c.corporation_markets
+		corporation_market = random.choice(c_markets)
```

Comment:
Nope. There is no reason to use the first one. c.f. before.
I believe we have different opinions on random and it usefulness for testing.